oasis/deployments.md

```diff
@@ -237,6 +242,67 @@ contact support via __Request help__ in the help menu.
    **Connections** will increase and you can view the connection details by
    clicking it.
 
+### Amazon Web Services (AWS)
+
+AWS offers a feature called [AWS PrivateLink](https://aws.amazon.com/privatelink){:target="_blank"}
+that enables you to privately connect your VPC to services, without exposure to
```

Comment:
please define the acronym the first time you use it.

Reply:
done